### PR TITLE
Unicode obfuscation hunting query

### PR DESCRIPTION
**Pull request by petebryan**
New Hunting Query looking for unicode in command lines which could be used to obfuscate malicious activity.

### CODE REVIEW

**Review by oshezaf — CHANGES_REQUESTED** [review]
Pete, can we add imProcessCreate as well? Do you want me to do that?

**Comment by petebryan** [comment]
> Pete, can we add imProcessCreate as well? Do you want me to do that?

Done

File: Hunting Queries/MultipleDataSources/UnicodeObfuscationInCommandLine.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  | extend CommandLine = translate('–”“', '-""', InitiatingProcessCommandLine)
+  | extend ASCII = isascii(CommandLine)
+  | where ASCII == false
+  | extend Account = AccountName, Computer = DeviceName),
+  (imProcessCreate
+  | where isnotempty(ActingProcessCommandLine)
+  | extend CommandLine = translate('–”“', '-""', ActingProcessCommandLine)
+  | extend ASCII = isascii(CommandLine)
+  | where ASCII == false
+  | extend Account = ActorUsernameType, Computer = DeviceName))
```
Comment (oshezaf) [inline Hunting Queries/MultipleDataSources/UnicodeObfuscationInCommandLine.yaml:38]: 1. Should be ActorUsername. Happy to discuss what those "*type" fields are about.
2. Should be DvcHostname, or just Dvc

**Review by oshezaf — CHANGES_REQUESTED** [review]
Small one

**Pull request merged**